plugins/opener/guest-js/init.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+import { invoke } from '@tauri-apps/api/core'
+
+// open <a href="..."> links with the API
+window.addEventListener('click', function (evt) {
+  if (evt.defaultPrevented || evt.button !== 0 || evt.metaKey || evt.altKey)
```

Comment:
We will probably need to consider middle mouse button as well

I'm think is it possible we handle this from a [new window handler](https://docs.rs/wry/latest/wry/struct.WebViewBuilder.html#method.with_new_window_req_handler) instead of doing it in js?

Reply:
looks like we may not be able to add middle mouse support atm, webview2 opens a new window and doesn't let our handler work, if we manage to add `new_window_handler` on Android, we can refactor this later.

Reply:
Yeah sure, as long as we can do it in a non breaking way I'm fine with it

Reply:
About middle click, you should be able to use [`onauxclick`](https://developer.mozilla.org/en-US/docs/Web/API/Element/auxclick_event)?

Reply:
Right, we can roll with this for now